packages/firebase_remote_config/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.2]
+
+* Fix iOS casting issue.
+* Bump Android library version.
```

Comment:
I would say something like "increase Android Firebase dependency to 15+" to be more specific about what you did here

Reply:
Done.